hstrat/genome_instrumentation/_HereditaryStratigraphicSurface.py

```diff
@@ -0,0 +1,301 @@
+from copy import deepcopy
+import operator
+import random
+from re import L
```

Comment:
for readability, let's do `import re` and then use `re.L` (`L` alone is confusing and hard to search)